kustomization.yaml

```diff
@@ -4,6 +4,6 @@ resources:
 - ansible-chatbot-deploy.yaml
 namespace: ansible-chatbot-stack
 images:
-- name: ansible-chatbot
-  newName: quay.io/ansible/ansible-chatbot
+- name: ansible-chatbot-stack
+  newName: quay.io/ansible/ansible-chatbot-stack
   newTag: aap-0.2.7
```

Comment:
Could you change this too please? We've dropped the `aap-` prefix (and it should probably be `0.2.9` now too?)